service/controller/v19/key/key.go

```diff
@@ -404,6 +409,10 @@ func RegionARN(customObject v1alpha1.AWSConfig) string {
 	return regionARN
 }
 
+func RequestedAvailabilityZones(customObject v1alpha1.AWSConfig) int {
```

Comment:
Can we try to establish some scheme here? Why not put spec and status in the function names?

Reply:
Renamed them to `{Spec,Status}AvailabilityZones()`